src/CoreBundle/Contao/Hooks/AbstractContentElementAndModuleCallback.php

```diff
@@ -381,7 +381,7 @@ public function saveCallback(string $value = null)
      */
```

Comment:
The `@return string` should now be changed to `@return string|null`

Reply:
done

Reply:
done